packages/react-theme-provider/src/tokensToStyleObject.test.ts

```diff
@@ -0,0 +1,44 @@
+import { tokensToStyleObject } from './tokensToStyleObject';
+
+describe('tokensToStyleObject', () => {
+  it('can convert flat tokens', () => {
+    expect(
+      tokensToStyleObject({
+        body: 'red',
+      }),
+    ).toEqual({
+      '--body': 'red',
+    });
+  });
+
+  it('can convert nested tokens', () => {
+    expect(
+      tokensToStyleObject({
+        body: {
+          fill: 'red',
+          text: 'blue',
+        },
+      }),
+    ).toEqual({
+      '--body-fill': 'red',
```

Comment:
I am a bit lean towards name the css variable camelCase (`--bodyFill` instead of `--body-fill`) bc these variable names cannot be minified.

Reply:
Let's chat about this;